app/assets/javascripts/discourse/widgets/default-notification-item.js.es6

```diff
@@ -154,6 +155,17 @@ export const DefaultNotificationItem = createWidget(
           );
         }
       });
+    },
+
+    mouseUp(e) {
```

Comment:
I think using `event` is much better, because it's way easier to grep for it in the codebase.